### PR TITLE
Release branch 2.16.0-b0

### PR DESCRIPTION
**Pull request by Pierre-Sassoulas**
## Description

Time to release the beta branch to get some feedback (without the latest astroid for now) ?

### CODE REVIEW

**Comment by DanielNoord** [comment]
Isn't the `astroid` merge the thing we want the most feedback one? It might actually help to get some reproducers for the `import-error`?

**Comment by DanielNoord** [comment]
Needs to be rebased and then can be merged!

File: doc/whatsnew/2/2.16/index.rst [hunk 1]
```diff
@@ -12,5 +12,23 @@
 Summary -- Release highlights
 =============================
 
+In 2.16.0 we added aggregation and composition understanding in ``pyreverse``, and a way to clear
+the cache in between run in server mode (originally for vs-code integration). Apart from the bug
```
Comment (DanielNoord) [inline doc/whatsnew/2/2.16/index.rst:16]:
```suggestion
the cache in between run in server mode (originally for the VS Code integration). Apart from the bug
```

File: doc/whatsnew/2/2.16/index.rst [hunk 2]
```diff
@@ -12,5 +12,23 @@
...
+fixes there's also a lot of new checks, and new extensions that have been asked for a long time
```
Comment (DanielNoord) [inline doc/whatsnew/2/2.16/index.rst:17]:
```suggestion
fixes there's also a lot of new checks, and new extensions that have been asked for for a long time
```

File: tbump.toml [hunk 3]
```diff
@@ -1,7 +1,7 @@
 github_url = "https://github.com/PyCQA/pylint"
 
 [version]
-current = "2.16.0-dev"
+current = "2.16.0-b0"
```
Comment (DanielNoord) [inline tbump.toml:4]:
```suggestion
current = "2.16.0b0"
```
As suggested by the relevant PEP:
https://peps.python.org/pep-0440/#pre-releases

File: pylint/__pkginfo__.py [hunk 4]
```diff
@@ -9,7 +9,7 @@
 
 from __future__ import annotations
 
-__version__ = "2.16.0-dev"
+__version__ = "2.16.0-b0"
```
Comment (DanielNoord) [inline pylint/__pkginfo__.py:12]:
```suggestion
__version__ = "2.16.0b0"
```

**Review by DanielNoord — CHANGES_REQUESTED** [review]
Couple of nits. I like the summary overview, well written!

File: tbump.toml (hunk 3 above)
Comment (Pierre-Sassoulas) [reply tbump.toml]: Nice catch I was going to release a defective version.

Comment (DanielNoord) [reply tbump.toml]: I don't think it matters, you can use what you want. But it's nice to follow "ecosystem standards" :smile:

Comment (DanielNoord) [reply tbump.toml]: Especially since we are a linter that enforces "ecosystem standards" :smile:

**Pull request merged**